doc/source/api/bobleesj.utils.sorters.rst

```diff
@@ -11,10 +11,10 @@ bobleesj.utils.sorters package
 Submodules
```

Comment:
This api ref is automatically generated by running `package create api-doc` in bobleesj.utils